composer.json

```diff
@@ -56,7 +56,7 @@
         "psalm/plugin-symfony": "^3.0",
         "sonata-project/doctrine-extensions": "^1.10.1",
         "symfony/browser-kit": "^4.4 || ^5.3 || ^6.0",
-        "symfony/debug": "^4.4 || ^5.3 || ^6.0",
+        "symfony/error-handler": "^4.4 || ^5.3 || ^6.0",
```

Comment:
Probably there are some other deps that need adjustment: psr/container needs to be added on 4.x, psr/cache needs to be removed on 5.x for example.

I am using:

https://github.com/maglnet/ComposerRequireChecker
https://github.com/composer-unused/composer-unused

both installed globally on my machine to test it (it is not possible to add them to dev kit atm, because they also give false positive or don't take into account optional deps)